dbschema/media.esdl

```diff
@@ -0,0 +1,59 @@
+module default {
+  type Image extending Media::VisualMedia {
+    overloaded _type: Media::Type {
+      constraint expression on (__subject__ = Media::Type.Image)
+    }
+  }
+  
+  type Video extending Media::VisualMedia, Media::TemporalMedia {
+    overloaded _type: Media::Type {
+      constraint expression on (__subject__ = Media::Type.Video)
+    }
+  }
+  
+  type Audio extending Media::TemporalMedia {
+    overloaded _type: Media::Type {
+      constraint expression on (__subject__ = Media::Type.Audio)
+    }
+  }
+}
+
+
+module Media {
+  scalar type Type extending enum<
+    Image,
+    Video,
+    Audio
+  >;
+  
+  abstract type MediaBase extending default::Resource {
```

Comment:
Not a Resource
```suggestion
  abstract type MediaBase {
```

Reply:
Ok, cool. Still trying understand what is and what isn't a resource. Thanks for the call out!